crates/egui/src/containers/window.rs

```diff
@@ -91,6 +93,12 @@ impl<'open> Window<'open> {
         self
     }
 
+    /// Only allow moving by dragging from the title bar.
+    pub fn movable_from_contents(mut self, movable: bool) -> Self {
+        self.move_title_bar_only = !movable;
```

Comment:
avoid negation in names

```suggestion
        self.movable_from_contents = movable;
```